packages/algorithm/package.json

```diff
@@ -59,9 +60,7 @@
     "terser": "^5.7.1",
     "tslib": "^2.4.0",
     "typedoc": "~0.23.9",
-    "typescript": "~4.7.3",
-    "webpack": "^5.74.0",
-    "webpack-cli": "^4.10.0"
```

Comment:
I'm guessing these are the only two webpack-related dependencies to delete.

Reply:
Yeah I agree